reactive/k8s.py

```diff
@@ -171,3 +212,21 @@ def docker_compose_kill_remove(service):
             # The docker-compose command to remove a service (forcefully).
             remove_command = 'docker-compose rm -f {0}'.format(service)
             check_call(split(remove_command))
+
+def start_services():
+    ''' Start all the required services for a Kubernetes cluster '''
+    with chdir('files/kubernetes'):
+        status_set('maintenance', 'Starting the kubernetes master container')
+        # Start the kubelet container that starts the three master services.
+        check_call(split('docker-compose up -d kubelet'))
+        set_state('kubelet.available')
+        # Start the proxy container
+        status_set('maintenance', 'Starting the kubernetes proxy container')
+        check_call(split('docker-compose up -d proxy'))
+        set_state('proxy.available')
+
+def recycle_kubernetes():
```

Comment:
I don't see where you call this method yet.  The point of the config-changed logic was to use the states to properly restart the services by removing the correct states.